src/components/Nav/NavComponent/LocationDropdown.js

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+
+const NavMockData = [
+  {
+    id: 1,
+    name: '신라스테이 서초',
+    city: '서울시',
+    gu: '서초구',
+    ro: '효령로',
+    detail: '427',
+  },
+  {
+    id: 2,
+    name: '힐튼 가든인',
+    city: '서울시',
+    gu: '서초구',
+    ro: '강남대로',
+    detail: '253',
+  },
+  {
+    id: 3,
+    name: '오클라우드 호텔',
+    city: '서울시',
+    gu: '서초구',
+    ro: '사평대로58길',
+    detail: '12',
+  },
+  {
+    id: 4,
+    name: '호텔 페이토 강남',
+    city: '서울시',
+    gu: '서초구',
+    ro: '강남대로',
+    detail: '259',
+  },
+  {
+    id: 5,
+
+    name: '어반플레이스 강남',
+    city: '서울시',
+    gu: '서초구',
+    ro: '',
+    detail: '',
+  },
+  {
+    id: 6,
+    name: 'The-K 호텔 서울    ',
+    city: '서울시',
+    gu: '서초구',
+    ro: '효령로77길',
+    detail: '30',
+  },
+];
+
+const LocationDropdown = () => {
+  const [hotels, setHotels] = useState(NavMockData);
+
+  return (
+    <StyledDropdown>
+      <StyledUl>
+        <StyledLi>
+          {hotels.map(hotel => {
```

Comment:
```suggestion
          {NavMockData.map(hotel => {
```
* setHotel에 의해서 변경해야 하는 데이터가 아니라면, state에 저장하지않고 구현하고자 하는 UI 부분에서 참조해서 map 메서드 구현하면 됩니다!